src/thor/map_matcher.cc

```diff
@@ -131,15 +131,7 @@ MapMatcher::FormPath(meili::MapMatcher* matcher,
                      const sif::TravelMode mode,
                      std::vector<std::pair<GraphId, GraphId>>& disconnected_edges,
                      Options& options) {
-  // Set costing based on the mode
```

Comment:
for some reason this file was set up like some old c99 non sense. i moved the variables to the places where they were most relevant.